scripts/scss-typography-generator.js

```diff
@@ -1,7 +1,7 @@
 const prefix = 'db';
 
 const fileHeader =
-	'@import "icon-family-calc";\n' +
+	'@use "variables" as *;\n@use "icon/icon-family-calc" as *;\n' +
```

Comment:
template literals might be easier to read here and at line 115

Reply:
fixed